service/controller/todo.go

```diff
@@ -0,0 +1,96 @@
+package controller
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/controller"
+	"github.com/giantswarm/operatorkit/informer"
```

Comment:
This has all changed 🙀 see https://github.com/giantswarm/app-operator/blob/b389fd674c5bee380d428c6ac3853375a656dad5/service/service.go for the new SOTA

Reply:
I have tried to make this work several times now but I can't make dep agree on the correct versions to use. How should this work?!

Reply:
Okay I think I made it work but needed to pin an operatorkit commit.